### ISSUE TITLE
/rest/V1/orders does not return shipping_address or shipping_address_id

### ISSUE THREAD

**Issue by ddeppner** [magento/magento2#3798]
## Steps to reproduce

```
1. Install rgranadino/mage2_vagrant
2. Run this from bash:
token=`curl -X POST "http://mage2.dev/rest/V1/integration/admin/token" \
     -H "Content-Type:application/json" \
     -d '{"username":"admin", "password":"password123"}' | cut -d\" -f2`
curl -X GET "http://mage2.dev/rest/V1/orders/1" \
     -H "Authorization: Bearer $token" \
     -H "Content-Type:application/json" |python -mjson.tool
```
## Expected result

I expect to be able to access the shipping_address_id and shipping_address on the order, just like I can with the billing_address_id and the billing_address.  shipping_address_id is clearly available in the sales_order table, but not exposed via the API.
## Actual result

The elements in question are not included in the json reply.  This means that I can't download an order for fulfillment in another system since there is no way to access the shipping address.

This also means I can't create a shipping record via the API that includes the shipping address from the original order.  The only option is to create a shipping record manually in the Admin interface and then updated it from external systems.  This is very klunky and labor intensive.
```json
{
    "base_currency_code": "USD",
    "base_discount_amount": 0,
    "base_discount_invoiced": 0,
    "base_discount_tax_compensation_amount": 0,
    "base_discount_tax_compensation_invoiced": null,
    "base_discount_tax_compensation_refunded": null,
    "base_grand_total": 34,
    "base_shipping_amount": 5,
    "base_shipping_discount_amount": 0,
    "base_shipping_discount_tax_compensation_amnt": null,
    "base_shipping_incl_tax": 5,
    "base_shipping_invoiced": 5,
    "base_shipping_tax_amount": 0,
    "base_subtotal": 29,
    "base_subtotal_incl_tax": 29,
    "base_subtotal_invoiced": 29,
    "base_tax_amount": 0,
    "base_tax_invoiced": 0,
    "base_to_global_rate": 1,
    "base_to_order_rate": 1,
    "base_total_due": 0,
    "base_total_invoiced": 34,
    "base_total_invoiced_cost": 0,
    "base_total_paid": 34,
    "billing_address": {
        "address_type": "billing",
        "city": "Calder",
        "country_id": "US",
        "customer_address_id": 1,
        "email": "roni_cost@example.com",
        "entity_id": 2,
        "firstname": "Veronica",
        "lastname": "Costello",
        "parent_id": 1,
        "postcode": "49628-7978",
        "region": "Michigan",
        "region_code": "Michigan",
        "region_id": 33,
        "street": [
            "6146 Honey Bluff Parkway"
        ],
        "telephone": "(555) 229-3326"
    },
    "billing_address_id": 2,
    "created_at": "2016-02-08 20:37:04",
    "customer_email": "roni_cost@example.com",
    "customer_firstname": "Veronica",
    "customer_gender": 0,
    "customer_group_id": 1,
    "customer_id": 1,
    "customer_lastname": "Costello",
    "discount_amount": 0,
    "discount_invoiced": 0,
    "discount_tax_compensation_amount": 0,
    "discount_tax_compensation_invoiced": null,
    "discount_tax_compensation_refunded": null,
    "entity_id": 1,
    "global_currency_code": "USD",
    "grand_total": 34,
    "increment_id": "000000001",
    "is_virtual": 0,
    "items": [
        {
            "amount_refunded": 0,
            "base_amount_refunded": 0,
            "base_discount_invoiced": 0,
            "base_discount_tax_compensation_amount": null,
            "base_discount_tax_compensation_invoiced": 0,
            "base_discount_tax_compensation_refunded": null,
            "base_price": 0,
            "base_row_invoiced": 0,
            "base_row_total": 0,
            "base_tax_invoiced": 0,
            "created_at": "2016-02-08 20:37:04",
            "discount_invoiced": 0,
            "discount_tax_compensation_amount": null,
            "discount_tax_compensation_canceled": null,
            "discount_tax_compensation_invoiced": 0,
            "discount_tax_compensation_refunded": null,
            "free_shipping": 0,
            "is_qty_decimal": 0,
            "item_id": 1,
            "name": "Iris Workout Top-XS-Red",
            "no_discount": 0,
            "order_id": 1,
            "original_price": 0,
            "price": 0,
            "product_id": 1420,
            "product_type": "simple",
            "qty_canceled": 0,
            "qty_invoiced": 1,
            "qty_ordered": 1,
            "qty_refunded": 0,
            "qty_shipped": 1,
            "row_invoiced": 0,
            "row_total": 0,
            "sku": "WS03-XS-Red",
            "store_id": 1,
            "tax_invoiced": 0,
            "updated_at": "2016-02-08 20:37:05",
            "weight": 1
        }
    ],
    "order_currency_code": "USD",
    "payment": {
        "account_status": null,
        "additional_information": [
            "Check / Money order",
            null,
            null
        ],
        "amount_ordered": 34,
        "amount_paid": 34,
        "base_amount_ordered": 34,
        "base_amount_paid": 34,
        "base_shipping_amount": 5,
        "base_shipping_captured": 5,
        "cc_last4": null,
        "entity_id": 1,
        "method": "checkmo",
        "parent_id": 1,
        "shipping_amount": 5,
        "shipping_captured": 5
    },
    "protect_code": "dfd28a",
    "quote_id": 1,
    "shipping_amount": 5,
    "shipping_description": "Flat Rate - Fixed",
    "shipping_discount_amount": 0,
    "shipping_discount_tax_compensation_amount": 0,
    "shipping_incl_tax": 5,
    "shipping_invoiced": 5,
    "shipping_tax_amount": 0,
    "state": "processing",
    "status": "processing",
    "status_histories": [],
    "store_currency_code": "USD",
    "store_id": 1,
    "store_name": "Main Website\nMain Website Store\n",
    "subtotal": 29,
    "subtotal_incl_tax": 29,
    "subtotal_invoiced": 29,
    "tax_amount": 0,
    "tax_invoiced": 0,
    "total_due": 0,
    "total_invoiced": 34,
    "total_item_count": 1,
    "total_paid": 34,
    "total_qty_ordered": 1,
    "updated_at": "2016-02-08 20:37:05",
    "weight": 1
}
```

**Comment by jhildreth** [comment]
I believe this is the same issue as #2628 , and according to #2861 will be fixed in 2.0.3.

**Comment by NadiyaS** [comment]
Hi @ddeppner,

thanks for reporting this issue. It is already being tracked by MAGETWO-46720 ticket.
And scheduled for 2.0.3 patch release.
This issue will be closed as duplicate of #2628 .
